lib/tests/test_lexer.c

```diff
@@ -128,53 +110,80 @@ _current_lloc(void)
 }
 
 #define assert_token_type(expected)                                     \
-  assert_gint(_current_token()->type, expected, "Bad token type at %s:%d", __FUNCTION__, __LINE__);
+  cr_assert_eq(_current_token()->type, expected, "Bad token type at %s:%d", __FUNCTION__, __LINE__);
 
-#define assert_parser_string(expected)                          \
-  _next_token();                                                        \
-  assert_token_type(LL_STRING);                                        \
-  assert_string(_current_token()->cptr, expected, "Bad parsed string at %s:%d", __FUNCTION__, __LINE__); \
+static void
+assert_parser_string(gchar *expected)
+{
+  _next_token();
+  assert_token_type(LL_STRING);
+  cr_assert_str_eq(_current_token()->cptr, expected, "Bad parsed string at %s:%d", __FUNCTION__, __LINE__);
```

Comment:
This change suffers the same expasion problem we discussed in assert_token_type. This should be a macro.

Reply:
ok. done